baanders2021reproducibility/journal/article.tex

```diff
@@ -8,5 +8,5 @@
 \renewcommand\emph[1]{{\bfseries #1}}
 \setlength\bibitemsep{0pt}
 \printbibliography
-
+\input{../openreview/appendix.tex}
```

Comment:
Since your appendix does not contain anything other than bibliography, please comment out this line. For ReScience journal, there is no need to specifiy the bibliography file, just add your references in the `bibliography.bib` file in journal folder.